lib/cli/gfx/index.js

```diff
@@ -0,0 +1,83 @@
+const 
+    blessed         =   require('blessed'),
+    pkg             =   require('../../../package.json'),
+    say             =   require('../../sayings/threads.json'),
+    serverUtil      =   require('./server-util')
+    template        =   require ('./template.js'),
+    ts              =   Date.now(),
+    date_ob         =   new Date(ts),
+    date            =   date_ob.getDate(),
+    month           =   date_ob.getMonth() + 1,
+    year            =   date_ob.getFullYear(),
+    currentdate     =   month + "/" + date + "/" + year,
+    space           =   ' ',
+    br              =   '\n',
+    header          =   '`COVID19-TRACKER-CLI (v'+pkg.version+')`',
+    tagline         =   '*A curl-based command line tracker for Novel Coronavirus or COVID-19 pandemic.*',
+    source          =   'Source: https://www.worldometers.info/coronavirus/',
+    repo            =   'Code: https://github.com/warengonzaga/covid19-tracker-cli',
+    bmcurl          =   'warengonza.ga/coffee4dev',
+    twitterhandle   =   '@warengonzaga',
+    twitterhashtag  =   '#covid19trackercli',
+    gcashNum        =   '+639176462753',
+    ansiBMC         =   '`(Buy Me A Coffee)` '+bmcurl,
+    ansiTwitter     =   twitterhandle+space+twitterhashtag,
+    ansiGCash       =   '(GCash) '+gcashNum;
```

Comment:
@WarenGonzaga We can extract these so it's reusable in other files.
@jkga I'm ok with this for now. Let's  have separate PR for this improvement